### PR TITLE
One-click deployment to Jelastic

### PR DESCRIPTION
**Pull request by sych74**
I have created a manifest that automatically deploys and configures TeamCity server in Jelastic.  

Installation dialog

Result

### CODE REVIEW

**Comment by sjoerdmulder** [comment]
Thanks but since this is a specific implementation detail related with commercial Jelastic offering i don't think it should be in this repository.

**Comment by sych74** [comment]
@sjoerdmulder thanks for the feedback. For sure, this automation is specifically written for Jelastic. However, there are benefits that any user can enjoy w/o buying anything - simplicity of the initial configuration (manual configuration is not trivial) and free cloud resources for testing, as result users can get an instant experience about TeamCity. There are several examples how others do it. 
https://github.com/ParsePlatform/parse-server-example
https://github.com/PokemonGoMap/PokemonGo-Map
https://github.com/xeb/fastText-docker

I would also recommend to add a similar deployment automation for other clouds, so it will be more neutral. But of course it's up to you.

File: manifest.jps [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		},
+		"env": {
+			"topology": {
+				"nodes": [{
+					"displayName": "TeamCity server",
+					"cloudlets": 16,
+					"nodeType": "docker",
+					"nodeGroup": "cp",
+					"docker": {
+						"image": "sych74/teamcity",
```
Comment (netflash) [inline manifest.jps:20]: shouldn't here be the image created by this repo - `sjoerdmulder/teamcity` ?

Comment (sych74) [reply manifest.jps:20]: You can replace my image by your own, but to make it work you need first to accept changes I sent you in my pull request above

Comment (sych74) [reply manifest.jps:20]: I mean we need to add small changes to docker-entrypoint.sh: define env var TEAMCITY_PORT and configure connection to database. After that we can change the image from this repo. Should I make two separate pull requests?

**Pull request closed without merging**